### PR TITLE
Phalconphp - Added 3.4 (version) and specific languages

### PR DESCRIPTION
**Pull request by niden**
Added 3.4 (version) and specific languages

Please review to ensure that the config is correct :)

### CODE REVIEW

File: configs/phalconphp.json [hunk 1]
```diff
@@ -1,7 +1,58 @@
...
+          "id-id",
+          "it-it",
+          "ja-jp",
+          "pl-pl",
+          "ru-ru",
+          "tr-tr",
+          "uk-ua",
+          "ur-in",
+          "zh-cn"
+        ]
```
Comment (s-pace) [inline configs/phalconphp.json]: Please add an extra_attributes for the version here just like https://github.com/algolia/docsearch-configs/blob/f3b40e5435fa5f289cfbd758eb5f4e4286b3f6a6/configs/brew_all.json#L10

File: configs/phalconphp.json [hunk 2]
```diff
@@ -33,4 +84,5 @@
     "761639486"
   ],
   "nb_hits": 499439
-}
```
Comment (s-pace) [inline configs/phalconphp.json]: Avoid this change please

**Comment by niden** [comment]
@s-pace I think I resolved the issues. Can you double check please?

File: configs/phalconphp.json (hunk 2 above)
Comment (niden) [reply configs/phalconphp.json]: Removed newline

File: configs/phalconphp.json (hunk 1 above)
Comment (niden) [reply configs/phalconphp.json]: Added the extra attributes

**Pull request merged**